radsat-sk/operation/crypt/crypt_globals.c

```diff
@@ -0,0 +1,12 @@
+#include "crypt_globals.h"
```

Comment:
Better file name? Why does this need it's own file? Will this always be here or is this just a placeholder until we get real values? If the latter, I suggest removing this from the PR. We can work on creating unit tests after finals or something